jobs/cronjob-ldap-group-sync.yml

```diff
@@ -159,6 +188,18 @@ parameters:
     displayName: "LDAP Bind User's password"
     description: "Password for the LDAP_BIND_DN user"
     required: true
+  - name: "LDAP_BIND_PASSWORD_SECRET"
+    displayName: "LDAP Bind Password Secret"
+    description: "The name for the secret in which to store the bind password"
+    value: "ldap-bind-password"
+    required: true
+  - name: "LDAP_CA_CERT"
+    displayName: "LDAP CA Certificate"
+    description: "The LDAP TLS Certificate Authority Certificate Bundle"
+  - name: "LDAP_CA_CERT_PATH"
+    displayName: "LDAP CA Certificate"
+    description: "The LDAP TLS Certificate Authority Certificate Bundle"
+    value: "/etc/config/ldap-ca.crt"
```

Comment:
Can we please default this to `''`?

Reply:
I feel defaulting this to empty string will make the template significantly more confusing to use.

If we really want to support insecure configuration as the default then I think we should have separate cronjob group-sync templates.